src/components/utils/seo.js

```diff
@@ -2,24 +2,42 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import Helmet from 'react-helmet'
 import { useStaticQuery, graphql } from 'gatsby'
-import defaultCard from '~images/card.png'
-import crdtCard from '~images/crdt-card.png'
 
-function SEO({ lang, meta, title, description, isRace = false }) {
-  const { site } = useStaticQuery(
+function SEO({ lang, meta, title, socialCard }) {
+  const { site, contentfulSocialCard } = useStaticQuery(
     graphql`
       query {
         site {
           siteMetadata {
             title
+          }
+        }
+        contentfulSocialCard(slug: { eq: "default" }) {
+          description {
             description
           }
+          image {
+            resize(width: 1200) {
+              src
+            }
+          }
         }
       }
     `,
   )
 
-  const card = isRace ? crdtCard : defaultCard
+  const defaultSocialCard = contentfulSocialCard
+  let description
+  let imageSrc
+  if (!socialCard) {
+    description = defaultSocialCard.description.description
+    imageSrc = defaultSocialCard.image.resize.src
+  } else if (socialCard.description && !socialCard.image) {
+    imageSrc = defaultSocialCard.image.resize.src
+  } else {
+    imageSrc = socialCard.image.resize.src
+    description = socialCard.description.description
+  }
 
```

Comment:
Maybe we need to normalize description a bit since Contentful is setting it as `description.description`, but we might pass it as `description` in a manual prop.

Reply:
Got it, would you like to restore the `description` prop or use the description value from the `socialCard` prop?

Reply:
Let's use the socialCard prop